### PR TITLE
Add missing information

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Hi @ajsb85 

In this PR I add the following changes

add professional services
change compatibility matrix
change build status

Thank you

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@Naylin15|1|

### References

Closes #N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,9 +93,13 @@ implementation project(':java-library-glpi')
 
 ## Matrix
 
-|**GLPI Version**|9.1|9.2|9.3|
-|:----|----|----|----|
-|**GLPI API Client**||||
+|**Library**|**Android API**|
```
Comment (ajsb85) [inline README.md]: You lost the format of the table.
How you can add a min and max values?
Could you provide an example?

File: README.md [hunk 2]
```diff
@@ -116,8 +122,15 @@ You can also chat with us via IRC in [#GLPI on freenode](http://webchat.freenode
...
 
+## Professional Services
+
+The GLPI Network services are available through our [Partner's Network](http://www.teclib-edition.com/en/partners/). We provide special training, bug fixes with editor subscription, contributions for new features, and more.
+
+Obtain a personalized service experience, associated with benefits and opportunities.
+
 ## Copying
 
+* **Name**: GLPI is a registered trademark of Teclib'.
```
Comment (ajsb85) [inline README.md]: The hyperlink to the Teclib edition website

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -106,6 +111,7 @@ In order to provide transparency on our release cycle and to maintain backward c
 
 See [the tags section of our GitHub project](https://github.com/glpi-project/java-library-glpi/tags) for changelogs for each release version. Release announcement posts on [the official Teclib' blog](http://www.teclib-edition.com/en/communities/blog-posts/) contain summaries of the most noteworthy changes made in each release.
```
Comment (ajsb85) [inline README.md]: See the tags section?
Why not the release section?
Reference: https://github.com/glpi-project/java-library-glpi/releases

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Naylin15) [reply README.md]: Link added 👍

File: README.md [hunk 4]
```diff
@@ -116,8 +122,15 @@ You can also chat with us via IRC in [#GLPI on freenode](http://webchat.freenode
...
 
+## Professional Services
+
+The GLPI Network services are available through our [Partner's Network](http://www.teclib-edition.com/en/partners/). We provide special training, bug fixes with editor subscription, contributions for new features, and more.
+
+Obtain a personalized service experience, associated with benefits and opportunities.
+
 ## Copying
 
+* **Name**: GLPI is a registered trademark of [Teclib'](http://www.teclib-edition.com/en/).
```
Comment (ajsb85) [inline README.md]: And the URL of the GLPI https://glpi-project.org/

File: README.md (hunk 1 above)
Comment (Naylin15) [reply README.md]: Here I switched the position of the version with the Android/Java column

**Review by ajsb85 — CHANGES_REQUESTED** [review]

**Comment by Naylin15** [comment]
Hi @ajsb85 

All changes made

Thank you

**Review by ajsb85 — APPROVED** [review]

**Pull request merged**